README.md

```diff
@@ -9,88 +9,38 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
 
 This image is ready to upload to akamai CDN the resource files. Contains the cdn-uploader script and his dependencies ready to run.
 
-Zero Step, define your environment variables:
+## Zero Step, define your environment variables: ##
 
-You need to define this variables in your host machine. They are needed to login with the CDN ftp.
-
-In Windows shell:
-
-```bash
-set AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
-set AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
-set AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
-set AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage
-```
-
-In Linux shell:
-
-```bash
-export AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
-export AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
-export AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
-export AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage, this is the root folder with write privileges inside the Akamai CDN
-```
-
-First Step, set the docker-compose.yml:
-
-The environment variables in the host machine are integrated in the docker image by the docker-compose.yml file. It is required by the cdn-uploader.js script.
+You need to define this environment variables. They are needed to login with the CDN ftp and cdn uploader script.
 
 ```bash
-version: '3.2'
-services:
-  app:
-    build:
-      context: .
-      dockerfile: editor.Dockerfile # Name of your dockerfile implementing doppler-docker-akamai-publish image
-    environment: 
-      - 'AKAMAI_CDN_HOSTNAME=${AKAMAI_CDN_HOSTNAME}'
-      - 'AKAMAI_CDN_USERNAME=${AKAMAI_CDN_USERNAME}'
-      - 'AKAMAI_CDN_PASSWORD=${AKAMAI_CDN_PASSWORD}'
-      - 'AKAMAI_CDN_CPCODE=${AKAMAI_CDN_CPCODE}'
+AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
+AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
+AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
+AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage
```

Comment:
I think that it is not necessary to define any variable, but at least these are being used in your example calling _docker run_, but you are not using _PROJECT_NAME_ and neither _VERSION_NAME_